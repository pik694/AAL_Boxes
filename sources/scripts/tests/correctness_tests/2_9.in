2
8 41 6
91 3 43

