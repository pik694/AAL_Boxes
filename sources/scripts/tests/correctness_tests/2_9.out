0 1 
13707->13707
